projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -654,23 +669,68 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     }
 
     /** @hidden @internal */
-    public handleClosing(event: CancelableEventArgs): void {
+    public handleClosing(event: CancelableBrowserEventArgs & IBaseEventArgs): void {
         this.onClosing.emit(event);
-    }
 
-    /** @hidden @internal */
-    public handleClosed(): void {
         if (this.value && this.value.start && !this.value.end) {
-            this.updateValue({ start: this.value.start, end: this.value.start });
+            this.value = { start: this.value.start, end: this.value.start };
         }
         if (this.value && !this.value.start && !this.value.end) {
-            this.updateValue(null);
+            this.value = null;
+        }
+
+        let outsideClick = true;
+        if (event.event) {
+            const elementRect: DOMRect = this.element.nativeElement.getBoundingClientRect();
```

Comment:
What's this? Whatever you are trying to avoid here, there must be a better way to check.

Reply:
fixed - if it is in drop down mode and click in the input it will not blur. In all other cases outside click will force blur